djangocms_navigation/utils.py

```diff
@@ -0,0 +1,26 @@
+import functools
+
```

Comment:
This is definitely not pep8 standard :P Both of the functools lines are standard libs so should be grouped together really